src/components/common/NotesTable/NoteUtils.js

```diff
@@ -1,44 +1,18 @@
-import styled from 'styled-components';
-import { Button, Input, Space } from 'antd';
+import { Badge, Button, Input, Space } from 'antd';
 import { SearchOutlined } from '@ant-design/icons';
+import * as MdIcons from 'react-icons/md';
+import '../styles/Notes.css';
 
-// Flag Styling
```

Comment:
I like this change of adding the styling into its own doc - much cleaner and easier to follow in my opinion